src/app/forms/custom-sections/weights/weights.component.ts

```diff
@@ -103,20 +103,23 @@ export class WeightsComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   private handleVehicleTechRecordChange(changes: SimpleChanges): void {
-    if (changes['vehicleTechRecord'] && !this.ladenWeightOverride && this.form) {
-      const { currentValue, previousValue } = changes['vehicleTechRecord'];
+    const { vehicleTechRecord } = changes;
+    if (this.form && vehicleTechRecord && !this.ladenWeightOverride) {
+      const { currentValue, previousValue } = vehicleTechRecord;
 
       const fieldsChanged = [
         'techRecord_seatsUpperDeck',
         'techRecord_seatsLowerDeck',
         'techRecord_manufactureYear',
-        'techRecord_grossKerbWeight',
-        'techRecord_standingCapacity'
+        'techRecord_grossKerbWeight'
```

Comment:
@Daniel-Searle can you review this? Think there maybe some logic that needs aligning with your changes, such as standing capacity and the manufacuterYear on the IF check